invenio_users_resources/records/api.py

```diff
@@ -37,6 +37,10 @@ def parse_user_data(user):
 
     data["preferences"].setdefault("visibility", "restricted")
     data["preferences"].setdefault("email_visibility", "restricted")
+    default_locale = current_app.config.get("BABEL_DEFAULT_LOCALE", "en")
```

Comment:
I think we should add invenio_i18n to explicit dependencies of this module, since we expect to use one of the babel config variables. ping @ntarocco @zzacharo @ppanero

Reply:
It is probably a good idea indeed.